### DISCUSSION TITLE
.NET: With `OpenTelemetry.Instrumentation.AWSLambda` is the Lambda Extension required?

### DISCUSSION THREAD

**Discussion by JohnYoungers** [open-telemetry/opentelemetry-lambda#1424] [Q&A]
Looking at the design proposal documentation: https://github.com/open-telemetry/opentelemetry-lambda/blob/main/docs/design_proposal.md

I'm trying to determine, for a .NET lambda properly using the `OpenTelemetry.Instrumentation.AWSLambda` library, the value of adding the extension:  If instead of including the layer, if I wire the exporter in my application to directly hit our target OTEL backend, am I losing any functionality?

**Comment by tylerbenson** [comment]
You might gain some benefit with using the collector lambda layer, but we don't have a lambda layer for .NET.

Specifically, the collector can capture additional details, and allow reporting the data without blocking the lambda.

**Reply by JohnYoungers** [reply]
Thank you! sorry, yes, I meant the collector lambda layer

**Reply by tylerbenson** [reply]
Unfortunately this is not a "well paved" process and I don't have experience with .NET, so please consider contributing back documentation on how to do so when you're done.
